app/database/queries/breeze_buddy/merchant_entities.py

```
@@ -0,0 +1,338 @@
+    if sort_order.lower() not in {"asc", "desc"}:
+        sort_order = "desc"
+
+    # Build queries
+    query = f"""
+        SELECT id, merchant_id, name, description, shop_identifiers,
+               is_active, owner_id, created_at, updated_at
+        FROM merchants
+        {where_clause}
+        ORDER BY {sort_by} {sort_order.upper()}
+        LIMIT ${param_idx} OFFSET ${param_idx + 1}
+    """
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)